### DISCUSSION TITLE
Why can't we set a default sort order?

### DISCUSSION THREAD

**Discussion by worldofgeese** [TechbeeAT/jtxBoard#1461] [Q&A]
I love the app but one thing that can be frustrating as a user is the inability to set a default sort order. When I open the app, I like to view entries in descending order of most recently modified. I've created and saved a filter but that filter is forgotten whenever I return to the app.

**Comment by patrickunterwegs** [comment]
Hey @worldofgeese , 
Basically the last used filters should be remembered. Also if you save a filter, when you reopen the app, it should take the last settings. 
Can you tell me how you open the app? Do you open it from the widget or through the app icon?

When you use the widget header, the app would open with the filter settings of the widget.

**Reply by worldofgeese** [reply]
Ah, this is the reason. I always open the app from the tasks widget and the widget used ascending order. I've now changed the widget's ordering.

**Reply by keunes** [reply]
I'm in the same situation here. I would love for the widget to have a different grouping/filtering/ordering than the main app, though. Reason: widget is small, so I filter out the less important tasks. However when opening the app (via the widget), however, I would prefer to get the full list.

@patrickunterwegs Would it make sense to open a feature request to a) disable the 'apply widget settings when opening main app' or b) explicitly set a default filter/grouping/ordering?
